docs/aws.md

```diff
@@ -0,0 +1,43 @@
+# Using LinuxKit on Amazon Web Services (AWS)
+
+This is a quick guide to run LinuxKit on AWS. 
+
+## Setup
+
+Before you proceed it's recommended that you set up the [AWS CLI](https://aws.amazon.com/cli/)
+and perform an `aws configure`
+
+If you do not wish to install these tools you should ensure that you set the AWS [environment variables](http://docs.aws.amazon.com/cli/latest/userguide/cli-environment.html)
+
+You will also need to create an Amazon S3 Storage Bucket for your LinuxKit images.
+
+Finally, you must set the `AWS_REGION` environment variable as this is used by the AWS Go SDK.
+```
+export AWS_REGION=eu-west-1
+```
+
+## Build an image
+
+AWS requires a `RAW` image. To create one:
+
+```
+$ moby build -output img examples/aws.yml
+```
+
+## Push image
+
+Do `linuxkit push aws -bucket bucketname aws.img` to upload it to the
+specified bucket, and create a bootable image from the stored image.
+
+Alternatively, you can use the `AWS_BUCKET` environment variable to specify the bucket name.
+
```

Comment:
I did by adding it to the pre-reqs here: https://github.com/linuxkit/linuxkit/pull/1964/files#diff-bde0f4c987d5959961ba607474757f2aR12